tests/multiple_database/tests.py

```diff
@@ -1583,7 +1583,7 @@ def test_pseudo_empty_fixtures(self):
         management.call_command('loaddata', 'pets', stdout=new_io, stderr=new_io)
         command_output = new_io.getvalue().strip()
         # No objects will actually be loaded
-        self.assertEqual(command_output, "Installed 0 object(s) (of 2) from 1 fixture(s)")
+        self.assertIn("Installed 0 object(s) (of 2) from 1 fixture(s)", command_output)
```

Comment:
If I'm not mistaken `command_output` shouldn't be altered since `verbosity != 3` in this case?